brandingbrand/flagship
# feat(fsweb): get pirateship working again on web

This implements a few fixes to get PirateShip working on web again:
 - Add babel.config to fsweb (current version of Babel doesn't appear to read from .babelrc anymore)
 - Add uglify webpack plugin (was bundled with prior versions of Webepack)
 - Add __DEFAULT_ENV__ to dev webpack config, defaults to production if not provided to keep behavior consistent with native
 - Fix compile-web script in PirateShip to properly send platform argument to Flagship init

**Test:**
 - `yarn ship:compile-web`
 - Navigate to localhost:3000
 - Switch the environment to Demandware, Shopify, or Mock to see data
